assignments/lambda-classes.js

```diff
@@ -1 +1,183 @@
 // CODE here for your Lambda Classes
+class Person {
+    constructor(attr) {
+        this.name = attr.name;
+        this.age = attr.age;
+        this.location = attr.location;
+        this.gender = attr.gender;
+    }
+    speak() {
+        const phrase = `Hello my name is ${this.name}, I am from ${this.location}.`
+        console.log(phrase)
+        return phrase;
+    }
+}
+
+class Instructor extends Person {
+    constructor(instAttr) {
+        super(instAttr);
+        this.specialty = instAttr.specialty;
+        this.favLanguage = instAttr.favLanguage;
+        this.catchPhrase = instAttr.catchPhrase;
+    }
+    demo(subject) {
+        const demoPhrase = `Today we are learning about ${subject}.`;
+        console.log(demoPhrase);
+        return demoPhrase;
+    }
+    grade(Student, subject) {
+        const gradePhrase = `${Student.name} receives a perfect score on ${subject}`;
```

Comment:
In JavaScript, general best practice is to use lower case variable names unless the variable is referring to a constructor object (like Instructor). This will not throw an error or anything, but it may confuse other developers.